### PR TITLE
create home page #16

### PR DESCRIPTION
**Pull request by ZinabHashem**

### CODE REVIEW

**Comment by ZinabHashem** [comment]
can you check

**Comment by diaahamad** [comment]
solve the conflicts

File: ui/src/app/core/components/contact-us/contact-us.component.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
+<div  fxLayoutAlign="center center" fxLayout="column">
```
Comment (diaahamad) [inline ui/src/app/core/components/contact-us/contact-us.component.html:1]: why this component in core module ?

File: ui/src/app/core/components/contact-us/contact-us.component.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  <h1>Contact Us</h1>
+  <div class="contact" style="background-image: url(assets/contact-background.png);">
```
Comment (diaahamad) [inline ui/src/app/core/components/contact-us/contact-us.component.html:3]: do not use inline style

Comment (ZinabHashem) [reply ui/src/app/core/components/contact-us/contact-us.component.html:3]: yes, but background image in sass need all path like"../../../../assets/contact-background.png"
when make it like "assets/contact-background.png" can't see it (make error).

File: ui/src/app/core/components/contact-us/contact-us.component.html (hunk 1 above)
Comment (ZinabHashem) [reply ui/src/app/core/components/contact-us/contact-us.component.html:1]: because it related with home page.

**Comment by ZinabHashem** [comment]
can you check

**Pull request merged**